app/policies/two_factor_authentication/backup_code_policy.rb

```diff
@@ -8,18 +8,6 @@ def configured?
       @user.backup_code_configurations.unused.any?
     end
 
-    def enabled?
```

Comment:
I see similar methods in `AuthAppPolicy` . Are they unused as well?

Reply:
Yep, thank you for catching that. I also removed a few others in the `policies` folder